### PR TITLE
[DOCS] Cookbook of how to modify placed order

### PR DESCRIPTION
**Pull request by arti0090**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.12         |
| License         | MIT                                                          |

### CODE REVIEW

File: docs/cookbook/administration/modify-placed-order.rst [hunk 1]
```diff
@@ -0,0 +1,111 @@
+How to modify placed order?
+===========================
+
+since time immemorial, people have been using trial and error methods to create or repeat processes that make their lives simpler.
+Thanks to this, we live in a time where everything is easily available and within reach.
+However, this time we will not focus on inventions, but on the mistakes that people still make.
+Usually, e-commerce processes go smoothly, but from time to time there will be a customer who confuses something in his order (e.g. the size of the shorts he wears).
+In this cookbook, I will try to show you how to easily fix this error as a store administrator.
+
+TLDR?
```
Comment (CoderMaggie) [inline docs/cookbook/administration/modify-placed-order.rst]:
```suggestion
TL;DR
```

File: docs/cookbook/administration/modify-placed-order.rst [hunk 2]
```diff
@@ -0,0 +1,111 @@
+How to modify placed order?
+===========================
+
+since time immemorial, people have been using trial and error methods to create or repeat processes that make their lives simpler.
```
Comment (CoderMaggie) [inline docs/cookbook/administration/modify-placed-order.rst]:
```suggestion
Since time immemorial, people have been using trial and error methods to create or repeat processes that make their lives simpler.
```

File: docs/cookbook/administration/modify-placed-order.rst [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+Thanks to this, we live in a time where everything is easily available and within reach.
+However, this time we will not focus on inventions, but on the mistakes that people still make.
+Usually, e-commerce processes go smoothly, but from time to time there will be a customer who confuses something in his order (e.g. the size of the shorts he wears).
+In this cookbook, I will try to show you how to easily fix this error as a store administrator.
```
Comment (CoderMaggie) [inline docs/cookbook/administration/modify-placed-order.rst]:
```suggestion
In this cookbook, we will try to show you how to easily fix this kind of mistake as a store administrator.
```

File: docs/cookbook/administration/modify-placed-order.rst [hunk 4]
```diff
@@ -0,0 +1,111 @@
+How to modify placed order?
+===========================
+
+since time immemorial, people have been using trial and error methods to create or repeat processes that make their lives simpler.
+Thanks to this, we live in a time where everything is easily available and within reach.
+However, this time we will not focus on inventions, but on the mistakes that people still make.
+Usually, e-commerce processes go smoothly, but from time to time there will be a customer who confuses something in his order (e.g. the size of the shorts he wears).
```
Comment (CoderMaggie) [inline docs/cookbook/administration/modify-placed-order.rst]:
```suggestion
Usually, e-commerce processes go smoothly, but from time to time there will be a customer who confuses something in his order (e.g. the size of the shorts they wear).
```

File: docs/cookbook/administration/modify-placed-order.rst [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+TLDR?
+-----
+
+Quick general steps of how to easily solve this issue:
+
+* From admin panel go to the order you want to edit
+* Click on the customer and go to their show page
+* Login as this customer whose order you want to edit
+* Place a new Order with the required modifications (like, change size of one of the products)
```
Comment (CoderMaggie) [inline docs/cookbook/administration/modify-placed-order.rst]:
```suggestion
* Place a new order with the required modifications (like changing the size of one of the products)
```

File: docs/cookbook/administration/modify-placed-order.rst [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+Step by step instruction
+------------------------
+
+If you have troubles of going with shorter instruction, here we will go step by step so you are sure of what to do:
+
+.. note::
+
+    The tutorial steps are done on Sylius demo environment under link: https://demo.sylius.com
+    If you want to test if Sylius suits you or try the steps, do not hesitate to check it out by yourself.
```
Comment (CoderMaggie) [inline docs/cookbook/administration/modify-placed-order.rst]:
```suggestion
    If you want to test if Sylius suits you or try the steps, do not hesitate to check it out on your own.
```

File: docs/cookbook/administration/modify-placed-order.rst [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+Impersonate as user and create an order
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's start with the steps that will let you impersonate as user and create new order as him/her.
```
Comment (CoderMaggie) [inline docs/cookbook/administration/modify-placed-order.rst]:
```suggestion
Let's start with the steps that will let you impersonate a user and create new order as them.
```

File: docs/cookbook/administration/modify-placed-order.rst [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+First login as an administrator into the admin panel and go to ``orders`` tab, and find the incorrect order (in this example Order #000000021):
```
Comment (CoderMaggie) [inline docs/cookbook/administration/modify-placed-order.rst]:
```suggestion
First, log in as an administrator into the admin panel, go to the ``orders`` tab, and find the incorrect order (in this example Order #000000021):
```

File: docs/cookbook/administration/modify-placed-order.rst [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+Now you need to find products with correct variants, add them and proceed to checkout
+
+.. image:: ../../_images/cookbook/administration/modify-placed-order/checkout.png
+    :align: center
+    :scale: 50%
+
+|
+
+On the next checkout steps just fill out the data that was used in old order,
+as payment method - use one of the offline methods (if your shop supports one),or proceed with any - and we will complete payment manually.
```
Comment (CoderMaggie) [inline docs/cookbook/administration/modify-placed-order.rst]:
```suggestion
as a payment method - use one of the offline methods (if your shop supports one), or proceed with any - and we will complete payment manually.
```

File: docs/cookbook/administration/modify-placed-order.rst [hunk 10]
```diff
@@ -0,0 +1,111 @@
+How to modify placed order?
+===========================
```
Comment (CoderMaggie) [inline docs/cookbook/administration/modify-placed-order.rst]:
```suggestion
How to edit orders in Sylius?
=============================
```

File: docs/cookbook/administration/map.rst.inc [hunk 11]
```diff
@@ -0,0 +1 @@
+* :doc:`/cookbook/administration/modify-placed-order`
```
Comment (CoderMaggie) [inline docs/cookbook/administration/map.rst.inc]:
```suggestion
* :doc:`/cookbook/administration/editing-orders`
```

**Review by CoderMaggie — APPROVED** [review]

File: docs/cookbook/administration/editing-orders.rst [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+
+Step by step instruction
+------------------------
+
+If you have troubles of going with shorter instruction, here we will go step by step so you are sure of what to do:
+
+.. note::
+
+    The tutorial steps are done on Sylius demo environment under link: https://demo.sylius.com
+    If you want to test if Sylius suits you or try the steps, do not hesitate to check it out by your own.
```
Comment (CoderMaggie) [inline docs/cookbook/administration/editing-orders.rst]:
```suggestion
    If you want to test if Sylius suits you or try the steps, do not hesitate to check it out on your own.
```

File: docs/cookbook/administration/editing-orders.rst [hunk 13]
```diff
@@ -0,0 +1,111 @@
...
+
+Impersonate as user and create an order
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's start with the steps that will let you impersonate an user and create new order as them.
```
Comment (CoderMaggie) [inline docs/cookbook/administration/editing-orders.rst]:
```suggestion
Let's start with the steps that will let you impersonate a user and create new order as them.
```

File: docs/cookbook/administration/editing-orders.rst [hunk 14]
```diff
@@ -0,0 +1,111 @@
...
+You can also add a note to it that it is modification of original order.
+In the end you will be greeted with message that your checkout has been completed.
+
+.. image:: ../../_images/cookbook/administration/editing-orders/completed-checkout.png
+    :align: center
+    :scale: 50%
+
+|
+
+Now lets go back to the customer page (where you can click ``show orders`` button) or orders page where you can search for just created order.
```
Comment (CoderMaggie) [inline docs/cookbook/administration/editing-orders.rst]:
```suggestion
Now let's go back to the customer page (where you can click the ``show orders`` button) or the orders page where you can search for the just-created order.
```

**Review by lchrusciel — APPROVED** [review]

File: docs/cookbook/administration/editing-orders.rst (hunk 13 above)
Comment (arti0090) [reply docs/cookbook/administration/editing-orders.rst]: @CoderMaggie `an` is correct here as:
"A" goes before words that begin with consonants. "An" goes before words that begin with vowels.
I might had school like 10 years or more ago, but "u" in user is a vowel 🙃

Comment (CoderMaggie) [reply docs/cookbook/administration/editing-orders.rst]: @arti0090 in 🇵🇱 yes, "u" is a vowel, but in English, when used at the beginning of a word - it is not :P

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "modify-placed-order"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by GSadee** [comment]
Thank you, @arti0090! :1st_place_medal:
